tests/unit/components/test.js

```diff
@@ -47,6 +55,16 @@ describe('Test', function() {
     it('should display the fail state', function() {
       expect(tree.textIn('.test-state')).to.be.equal(failTest.state);
     });
+
+    it('should pass the failDetails to the Details component', function() {
+      var expected = {
+        baseline: failTest.result.baseline,
+        diff: failTest.result.diff,
+        screenshot: failTest.result.screenshot
+      };
+
+      expect(tree.findNode('Details').props.paths).to.be.deep.equal(expected);
```

Comment:
You're checking the baseline here and you're checking it again in one of the above tests. This tells me something: that you're tests are not focused, they test more than they can chew. So, write a test for a single property. You only need to check the baseline once (in the pass test describe).